src/app/(components)/Navbar.tsx

```diff
@@ -0,0 +1,14 @@
+import Menu from "./Menu"
+import "./Navbar.css" 
+const Navbar = ()=>{
+    return(
+        <div className="navbarContainer">
+            <Menu/>  
```

Comment:
You navbar should display not only the menu, but also the logo and the cart component

Reply:
not sure if i understand this one

Reply:
You did well here, Creating the Cart component and using an image. Next,js comes with an Image component that optimizes the images for you. Let's use it